ansible/roles/tenderdash/templates/tenderdash/node_key.json.j2

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Seed nodes need this?

Reply:
I don't know, I just combined the dashmate/mn_evo_services deploy methods to achieve this

Reply:
This one is needed for validators only. Let's try without it?

Reply:
Works fine without it